### PR TITLE
Fixed for issues #511 and #515

### PR DESCRIPTION
**Pull request by sprainbrains**
I fixed  a height for command line interface in a fullscreen mode for Windows 10. (look  issues: [1](https://github.com/equalsraf/neovim-qt/issues/511) and [2](https://github.com/equalsraf/neovim-qt/issues/515)).

### CODE REVIEW

File: src/gui/mainwindow.cpp [hunk 1]
```diff
@@ -4,342 +4,348 @@
...
-		m_shell->resizeNeovim(m_shell->size());
-	}
+    // Neovim finished resizing, resize it back to the actual
+    // widget size - this avoids situations when neovim wants a size that
+    // exceeds the available widget size i.e. the GUI tells neovim its
+    // size, not the other way around.
+    if (isMaximized() || isFullScreen()) {
+        QSize size = geometry().size();
+        if (m_tabline_bar->isVisible()) {
+            size.scale(size.width(),
```
Comment (equalsraf) [inline src/gui/mainwindow.cpp]: The scale here is unnecessary right? You can just `size.setHeight(size.height() - m_tabline_bar->geometry().size().height())`

**Comment by equalsraf** [comment]
Hi @sprainbrains. thanks for the fix.

Can you keep the style formatting changes out, the make the diffs rather hard to read.

**Comment by sprainbrains** [comment]
> Hi @sprainbrains. thanks for the fix.
> 
> Can you keep the style formatting changes out, the make the diffs rather hard to read.

Sorry for that. I forgot to disable autoformatter in may editor.

File: src/gui/mainwindow.cpp (hunk 1 above)
Comment (sprainbrains) [reply src/gui/mainwindow.cpp]: > The scale here is unnecessary right? You can just `size.setHeight(size.height() - m_tabline_bar->geometry().size().height())`

You right. Enough to add this code to mainwindow.cpp : 

`if (m_tabline_bar->isVisible()) {
            size.setHeight(size.height() - m_tabline_bar->geometry().size().height());
 }`

**Pull request merged**